newLib.js

```diff
@@ -5,13 +5,15 @@ const projectName = process.argv.slice(2).toString()
 
 const execAsync = util.promisify(cp.exec)
 
+const isReact = process.env.IS_REACT ? 'react' : 'lib'
+
 async function execute(command) {
   const logs = await execAsync(command)
   process.stdout.write(logs.stdout)
 }
 
 const main = async () => {
-  await execute(`npx hygen lib new ${projectName}`)
+  await execute(`npx hygen ${isReact} new ${projectName}`)
   await execute(`npx lerna create ${projectName}  --yes`)
```

Comment:
I think as you are already pre-creating the package folder with hygen `lerna create` is not even necessary anymore as it's just creating the package.json 🤔 probably you can use just have a package template in hygen and then we don't even need newLib.js anymore

Reply:
:thinking: but some fields can be a bit difficult to add with a hygen templat  (like author)

Reply:
what do you think @fakenickels ?

Reply:
ah but the author can be `Astrocoders <contact@astrocoders.com>` always!